examples/nftvm/storage/nft_storage.go

```diff
@@ -0,0 +1,273 @@
+// Logic relevant to reading/writing NFTs to storage
+package storage
+
+import (
+	"context"
+	"encoding/binary"
+	"errors"
+
+	"github.com/ava-labs/avalanchego/database"
+	"github.com/ava-labs/hypersdk/codec"
+	"github.com/ava-labs/hypersdk/consts"
+	"github.com/ava-labs/hypersdk/state"
+	"github.com/ava-labs/hypersdk/utils"
+
+	mconsts "github.com/ava-labs/hypersdk/examples/nftvm/consts"
+)
+
+// PREFIXES MUST START AFTER 0x3 TO AVOID CONFLICTS
+const (
+	nftCollectionPrefix    = 0x4
+	nftInstancePrefix      = 0x5
+	marketplaceOrderPrefix = 0x6
+)
```

Comment:
Reading this creates a mental load because now I need to think about where the other prefixes are being used and if there are any prefixes after this.

I'd recommend defining the DB layout and all of the prefixes in a single file and then if you want to split up the functions, dividing those up.